### PR TITLE
Add attribute times to the docstring of Raw, Epochs and Evocked.

### LINKED ISSUE #4700: Undocumented times attribute in Raw and Epoch object
Both `Raw` and `Epoch` classes have a `times` attribute. However, this attribute is not described in the documentation (see [epoch](https://martinos.org/mne/stable/generated/mne.Epochs.html) and [raw](https://martinos.org/mne/stable/generated/mne.io.Raw.html)).

(I can do a PR).

### PR DESCRIPTION
**Pull request by aweinstein**
Fix #4700.

### CODE REVIEW

File: mne/io/fiff/raw.py [hunk 1]
```diff
@@ -61,6 +61,8 @@ class Raw(BaseRaw):
         List of channels' names.
     n_times : int
         Total number of time points in the raw file.
+    times :  ndarray
+        Time vector.
```
Comment (agramfort) [inline mne/io/fiff/raw.py]: can you specify it is in seconds and that it starts from 0. independently of first_samp attrubute?

File: mne/evoked.py [hunk 2]
```diff
@@ -91,6 +91,8 @@ class Evoked(ProjMixin, ContainsMixin, UpdateChannelsMixin,
         Array of time instants in seconds.
     data : array of shape (n_channels, n_times)
         Evoked response.
+    times :  ndarray
+        Time vector.
```
Comment (agramfort) [inline mne/evoked.py]: can you specify it is in seconds and that it goes from tmin to tmax.

File: mne/epochs.py [hunk 3]
```diff
@@ -1913,6 +1913,8 @@ class Epochs(BaseEpochs):
         or 'USER' for user-defined reasons (see drop method).
     filename : str
         The filename of the object.
+    times :  ndarray
+        Time vector.
```
Comment (agramfort) [inline mne/epochs.py]: can you specify it is in seconds and that it goes from tmin to tmax.

**Comment by aweinstein** [comment]
@agramfort Improved the descriptions.

**Comment by agramfort** [comment]
thx @aweinstein

**Pull request merged**